legal-api/src/legal_api/services/authz.py

```diff
@@ -237,21 +226,24 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
             }
         },
         Business.State.HISTORICAL: {
+            'alteration': {
+                'legalTypes': ['BC', 'BEN', 'ULC', 'CC']
+            },
```

Comment:
A business cannot file an alteration filing when business is historical.  Please remove.